### PR TITLE
Task 1 of lesson 1and 2

### PR DESCRIPTION
**Pull request by chidiprosper07**
**_i have finished my task one come and review thanks_**@ChidiPatrick @Promise-J

### CODE REVIEW

**Review by Pieta9221 — COMMENTED** [review]
🌟🌟Good job!

File: html/index.html [hunk 1]
```diff
@@ -1,50 +1,77 @@
...
-    </header>
-    <nav>
-        <a href="#">HOME</a>
-        <a href="#">ABOUT</a>
-        <a href="#">CONTACT US</a>
-    </nav>
-    <section>
-        <h1>MISSION STATEMENT</h1>
-        <p>To Deliver Value To All Stakeholders Including Shareholders, Employees and Our Host Communities Through High Yield On Investment, Employee Growth, Corporate Responsibility, Operational Excellence, Cutting-Edge Technology and Continual Civic Engagement.</p>
-        <h1>VISSION STATEMENT</h1>
```
Comment (Pieta9221) [inline html/index.html]: inaccurate spelling for Vision

File: html/index.html [hunk 2]
```diff
@@ -1,50 +1,77 @@
...
-        <p>Our Vision Is To Become The Number One Player In The Agro-Allied Processing Industry
-            (Tomato And Cassava) In Nigeria And West Africa.</p>
-            <img src="../image/tomato.jfif" alt="tomato photo">
-            <h1>Ekcel Farms Limited (Ekcel Farms) is a wholly owned subsidiary of Oilserv Limited. The thought to diversify into the Agricultural sector of the economy was the brainchild of an astute entrepreneur and industrialist – Engr. (Sir) Emeka Okwuosa. The project is aimed at facilitating a massive agricultural revolution in Nigeria and the sub-region which in return foster an enormous economic transformation through employment generation, elimination of agricultural losses due to the current shortfall in processing facilities and encourage independent farming through an organised out-growers scheme.
```
Comment (Pieta9221) [inline html/index.html]: only one h1 tag is required within the section tag and it has been implemented on line 17. Take a look at your task and accurately follow the laid down guidelines.

**Review by Pieta9221 — COMMENTED** [review]
😢 kindly make the necessary changes before merging

**Review by Pieta9221 — COMMENTED** [review]
I do not think adding an extra image was necessary as it was not in use.

**Review by promise-J — APPROVED** [review]
Cleared. Very good. 

You can now merge, Mr. Prosper

**Pull request merged**